packages/@react-spectrum/button/src/Button.tsx

```diff
@@ -54,7 +54,7 @@ function Button<T extends ElementType = 'button'>(props: SpectrumButtonProps<T>,
   props = useSlotProps(props, 'button');
   props = disablePendingProps(props);
   let {
-    elementType: ElementType = 'button',
+    elementType: Element = 'button',
```

Comment:
This is to accommodate the docs generator. It was getting confused between `ElementType` declared here and the one used in the type parameter. Found by the new console.log added in DocsPackager.